pi-hole.json

```
@@ -60,7 +60,7 @@
                 }
             }
         },
-        "description": "DNS Server that acts as a black hole for Internet advertisements.<p>Based on the official Pi-Hole docker image: <a href='https://hub.docker.com/r/pihole/pihole' target='_blank'>https://hub.docker.com/r/pihole/pihole</a>.",
+        "description": "DNS Server that acts as a black hole for Internet advertisements.<p>Based on the official docker image: <a href='https://hub.docker.com/r/pihole/pihole' target='_blank'>https://hub.docker.com/r/pihole/pihole</a>, available for amd64 architecture only.</p>",
```

```suggestion
        "description": "DNS Server that acts as a black hole for Internet advertisements.<p>Based on the official docker image: <a href='https://hub.docker.com/r/pihole/pihole' target='_blank'>https://hub.docker.com/r/pihole/pihole</a>, available for amd64 and arm64 architecture.</p>",
```
@Hooverdan96 this is another one that was hidden in a drop down I think.